common/code_spelling_ignore_words.txt

```diff
@@ -569,6 +569,7 @@ gotperspective
 gotrootobject
 gpo
 gracefulshutdown
+graphql
```

Comment:
this one is for pylint..

the one you need to fix is:
https://github.com/buildbot/buildbot/blob/master/master/docs/spelling_wordlist.txt#L1